doc/classes/Color.xml

```diff
@@ -241,38 +245,42 @@
 			<return type="bool" />
 			<param index="0" name="color" type="String" />
 			<description>
-				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. [param color] is not case sensitive, and may be prefixed with a '#' character.
-				For a string to be valid it must be three-digit or six-digit hexadecimal, and may contain an alpha channel value.
+				Returns [code]true[/code] if [param color] is a valid HTML hexadecimal color string. The string must be a hexadecimal value (case-insensitive) of either 3, 4, 6 or 8 digits, and may be prefixed by a hash sign ([code]#[/code]). This method is identical to [method String.is_valid_html_color].
 				[codeblocks]
 				[gdscript]
-				var result = Color.html_is_valid("#55aaFF") # result is true
-				result = Color.html_is_valid("#55AAFF20") # result is true
-				result = Color.html_is_valid("55AAFF") # result is true
-				result = Color.html_is_valid("#F2C") # result is true
-				result = Color.html_is_valid("#AABBC) # result is false
-				result = Color.html_is_valid("#55aaFF5") # result is false
+				Color.html_is_valid("#55aaFF")   # Returns true
+				Color.html_is_valid("#55AAFF20") # Returns true
+				Color.html_is_valid("55AAFF")    # Returns true
+				Color.html_is_valid("#F2C")      # Returns true
+
+				Color.html_is_valid("#AABBC)     # Returns false
+				Color.html_is_valid("#55aaFF5")  # Returns false
 				[/gdscript]
 				[csharp]
-				var result = Color.HtmlIsValid("#55AAFF"); // result is true
-				result = Color.HtmlIsValid("#55AAFF20"); // result is true
-				result = Color.HtmlIsValid("55AAFF); // result is true
-				result = Color.HtmlIsValid("#F2C"); // result is true
-				result = Color.HtmlIsValid("#AABBC"); // result is false
-				result = Color.HtmlIsValid("#55aaFF5"); // result is false
+				// This method is not available in C#. Use `String.IsValidHtmlColor()`.
```

Comment:
I would prefer `StringExtensions.IsValidHtmlColor()` since the method is not in `System.String`, it's in the `StringExtensions` class. You can use the method as if it was an instance method of `System.String` because it is defined as an extension method.

An extension method is basically a static method that has syntax sugar so it can be used as if it was an instance method of the first parameter.